lib/Raven/Client.php

```diff
@@ -16,7 +16,7 @@
 
 class Raven_Client
 {
-    const VERSION = '1.10.x-dev';
+    const VERSION = '1.11.x-dev';
```

Comment:
We forgot to fix this on the tag, right?

Reply:
No? https://github.com/getsentry/sentry-php/blob/1.10.0/lib/Raven/Client.php#L19

Reply:
Oh you did it without a PR! :+1: Missed it

Reply:
Yeah it can be done on the release branch itself so don’t need to PR to do things over there :eyes: